app/src/features/surveys/view/survey-animals/TelemetryDeviceForm.tsx

```diff
@@ -204,13 +219,13 @@ const DeviceFormSection = ({ values, index, mode, removeAction }: IDeviceFormSec
           <Grid item xs={6}>
             <TelemetrySelectField
```

Comment:
The keyx / cfg upload functionality for Vectronic and Lotek devices has disappeared when adding a new device, but still shows up on edit